app/views/content/start-teacher-training-this-september.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "There's still time to apply"
+subtitle: "Start teacher training this September"
+description: |-
+  There's still time to apply for teacher training in 2021. You could apply now and be training by September 2021 if you follow these instructions.
+date: "2021-06-07"
+image: "media/images/content/teaching-as-a-career-hero-dt.jpg"
+mobileimage: "media/images/content/teaching-as-a-career-hero-mob.jpg"
+backlink: "../"
+keywords:
+    - Application
+    - Apply
+    - 2021 
+    - support
+    - training
+---
+Everyday, people are being accepted onto courses that start this September. 
+
+Places fill up quickly at this time of year, but there are still hundreds of courses available across the country.
+
+Courses cover a range of subjects including modern languages, mathematics, science and more.
+
+## Find a course
+
+Search for a teacher training course:
+
+<a class ="button button--white" href ="https://www.find-postgraduate-teacher-training.service.gov.uk">Find postgraduate teacher training</a>
```

Comment:
To me it feels like this section is a bit short. The heading, paragraph and button text all mirror each other. Is it worth moving the "Courses cover.." paragraph from just above into it and perhaps expanding on it a bit?